packages/frontend/@n8n/i18n/src/locales/en.json

```diff
@@ -2821,6 +2822,10 @@
 	"dataStore.add.input.name.label": "Data Store Name",
 	"dataStore.add.input.name.placeholder": "Enter data store name",
 	"dataStore.add.error": "Error creating data store",
+	"dataStore.delete.confirm.title": "Delete data store",
```

Comment:
<img width="206" height="141" alt="image" src="https://github.com/user-attachments/assets/f5ff89b8-05fc-4e46-b2f5-7fb5d690a4ce" />

the capitalization of `dataStore.add.button.label` is different to `projects.header.create.credential`, which looks a bit odd 🤔